### PR TITLE
Config to docs run

### PR DESCRIPTION
**Pull request by mmattel**
Referencing #704 

Only merge when https://github.com/owncloud/core/pull/40473 (Add a genrenic config sample description for S3 Primary) has been merged.

Backport to 10.11 and 10.10

### CODE REVIEW

File: modules/admin_manual/pages/configuration/server/config_apps_sample_php_parameters.adoc [hunk 1]
```diff
@@ -516,6 +516,28 @@ Please note, only one group can be defined. Default = empty = no restriction.
...
+The detailed configuration of that array depends on the S3 provider.
+This example can therefore only show the general construct.
+See the "S3 Compatible Object Storage as Primary Storage Location" documentation for more details.
+
+==== Code Sample
+
+[source,php]
+....
+'objectstore' => [
+	'class' => 'Implementation\Of\OCP\Files\ObjectStore\IObjectStore',
```
Comment (jnweiger) [inline modules/admin_manual/pages/configuration/server/config_apps_sample_php_parameters.adoc:534]:
```suggestion
	'class' => 'OCA\Files_Primary_S3\S3Storage',
```

How about a real world string there? -- I see it as a fixed string. Is there a reason to consider it as a variable? The only other implementation I know of is objectstore -- which is deprecated.

**Review by jnweiger — CHANGES_REQUESTED** [review]
see https://github.com/owncloud/core/pull/40473#pullrequestreview-1168283615

File: modules/admin_manual/pages/configuration/server/config_apps_sample_php_parameters.adoc (hunk 1 above)
Comment (mmattel) [reply modules/admin_manual/pages/configuration/server/config_apps_sample_php_parameters.adoc:534]: See my answer in core

**Review by jnweiger — APPROVED** [review]
yes

**Review by phil-davis — APPROVED** [review]

**Pull request merged**